### PR TITLE
Remove cache volume and add postStart cache warmup to the deployment

### PR DESCRIPTION
**Pull request by Ginungagap**

### CODE REVIEW

File: mautic/templates/deployment.yaml [hunk 1]
```diff
@@ -54,6 +54,10 @@ spec:
               value: {{ .Values.runCronJobs | quote }}
             - name: MAUTIC_TRUSTED_PROXIES
               value: {{ .Values.trustedProxies | quote }}
+          lifecycle:
+            postStart:
+              exec:            
+                command: ["/bin/sh", "-c", "php /var/www/html/bin/console cache:warmup && chown www-data.www-data -R /var/www/html/var"]
```
Comment (R3VoLuT1OneR) [inline mautic/templates/deployment.yaml:60]: Why not `sudo -u www-data php /var/www/html/bin/console cache:warmup`?

Comment (Ginungagap) [reply mautic/templates/deployment.yaml]: I have added "sudo -u www-data"

Comment (R3VoLuT1OneR) [reply mautic/templates/deployment.yaml]: So remove the "&& chown www-data.www-data -R /var/www/html/var"

Comment (R3VoLuT1OneR) [reply mautic/templates/deployment.yaml]: So remove the "&& chown www-data.www-data -R /var/www/html/var"

**Pull request merged**